org.eclipse.buildship.kotlinDSL.provider/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
+    id 'application'
+}
```

Comment:
This is an eclipse plugin project; you don't need to apply the application plugin,

```suggestion
```